tests/requirements.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- src: http://bitbucket.int.sys.idealista/scm/as/mysql-role.git
+- src: https://github.com/idealista/mysql-role
```

Comment:
This is nice, but it's better to use ansible galaxy: idealista.mysql-role
version: 1.2.0
no scm :)